CHANGES.md

```diff
@@ -37,9 +41,14 @@ To be released.
 
 ### CLI tools
 
+  - Added `planet store migrate-index` for index database migration.
+    (from column families based to key-prefix)  [[#1838]]
+
 [#1828]: https://github.com/planetarium/libplanet/issues/1828
 [#1831]: https://github.com/planetarium/libplanet/pull/1831
 [#1832]: https://github.com/planetarium/libplanet/pull/1832
+[#1838]: https://github.com/planetarium/libplanet/pull/1838
+[column families]: https://github.com/facebook/rocksdb/wiki/Column-Families
```

Comment:
This ref is available? I mean, I can't find on the upper comment

Reply:
It is referenced in https://github.com/planetarium/libplanet/pull/1838/files#diff-d975bf659606195d2165918f93e1cf680ef68ea3c9cab994f033705fea8238b2R23